### ISSUE TITLE
Feature Request: New data visualizations

### ISSUE THREAD

**Issue by asaprahul** [caesarHQ/textSQL#38]
Currently, the project only supports visualizing zipcodes and cities on the map using Mapbox
But data can be visualized in many other interesting ways like Bar Charts, Heatmaps and Pie charts.

Not every kind of data can be (or should be) visualized on a map. 

For example, a query like "What percent of total crime in San Francisco is burglary vs in New York City" is perfect for visualizing as a stacked bar chart, but really hard to visualize on map.
TLDR: we need to support more visualizations!

![image](https://user-images.githubusercontent.com/10172332/224442915-ccb31cbb-1689-4a85-ab4c-6ab1166b8a99.png)

![image](https://user-images.githubusercontent.com/10172332/224442978-21fe0ea0-4c19-4441-971a-47d43b457c94.png)

Plotly looks promising: https://plotly.com/javascript/bar-charts/

**Comment by asaprahul** [comment]
Done. Will punt heatmap for later, until users ask for it.
